int8 fromPlayer
int8 playerID
---
bool threw